src/featurestorebundle/widgets/WidgetsFactory.py

```diff
@@ -37,19 +40,13 @@ def create_for_target(self):
         self.__widgets.add_text("number_of_time_units", self.__defaults.number_of_time_units)
 
     def create_target_name(self):
+        targets = [
+            row.__getattr__(get_target_id_column_name())
```

Comment:
why not `getattr(row, get_target_id_column_name())` ?

Reply:
fair point